include/rtc/websocket.hpp

```diff
@@ -46,6 +46,8 @@ class RTC_CPP_EXPORT WebSocket final : private CheshireCat<impl::WebSocket>, pub
 		bool disableTlsVerification = false; // if true, don't verify the TLS certificate
 		optional<ProxyServer> proxyServer;   // unsupported for now
 		std::vector<string> protocols;
+		int maxMissedPongsAllowed = 2;                   // -1 is no check
```

Comment:
I prefer using an `optional<int>` to allow disabling, which should be unset by default to keep the current behavior.